### PR TITLE
Solves issue#1

### LINKED ISSUE #1: Add config to hyperparameters json - YOLOv3
Add the config in relevant positions at hyperparameters.json (don't parse it yet) just structure the hyperparameters.json. Link to config file:- https://github.com/aladdinpersson/Machine-Learning-Collection/blob/master/ML/Pytorch/object_detection/YOLOv3/config.py

### PR DESCRIPTION
**Pull request by Aayush-99**
Closes  #1
- Added yolov3 hyperparameters
- learning rate and weight decay added to optimizer
- epochs, confidence threshold, mean average precision IOU threshold and non max suppression iou threshold added to training
- batch size and image size added to dataset arguments

### CODE REVIEW

**Review by frankhart2018 — CHANGES_REQUESTED** [review]
1. Update branch name to in-1
2. I don't understand the PR title, please make sure it is related to the PR.
3. I don't think the dataloader and dataset classes are correct, anyway it wasn't included in this issue, it is part of #2 which I am doing.
4. I don't see any transforms file in the object_detection submodule.
